shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.test.js

```diff
@@ -14,6 +14,12 @@ describe('generatePrintablePendingReportInteractor', () => {
   let mockUser;
 
   const mockFoundDocuments = [
+    {
```

Comment:
I'd love to see a specific named test for this bug so that if it fails again it serves as documentation for the expected behavior!

Reply:
We added additional out of order items to the mockFoundDocuments array. We also better named an existing test that was already testing the formatting and sorting of the pendingItems. This test also now accounts for the additional out of order mocked items as well.